aws/aws.go

```diff
@@ -177,8 +191,21 @@ func NukeAllResources(account *AwsAccountResources, regions []string) error {
 
 		resourcesInRegion := account.Resources[region]
 		for _, resources := range resourcesInRegion.Resources {
-			if err := resources.Nuke(session); err != nil {
-				return errors.WithStackTrace(err)
+			if len(resources.ResourceIdentifiers()) >= 500 {
+				logging.Logger.Infoln("Terminating Resources in batches")
+				batches := split(resources.ResourceIdentifiers(), 200)
+
+				for _, batch := range batches {
+					if err := resources.NukeBatch(session, batch); err != nil {
```

Comment:
It feels a bit weird to have a nearly identical `NukeBatch` and `Nuke` method in each resource type. Perhaps there should solely be a `Nuke` method that takes a list of identifiers as a parameter?